tests/rt.sh

```diff
@@ -400,9 +400,9 @@ if [[ $SINGLE_NAME != '' ]]; then
 fi
 
 if [[ $MACHINE_ID = hera.* ]] || [[ $MACHINE_ID = orion.* ]] || [[ $MACHINE_ID = cheyenne.* ]] || [[ $MACHINE_ID = jet.* ]]; then
-  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201019/${RT_COMPILER^^}}
+  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201022/${RT_COMPILER^^}}
 else
-  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201019}
+  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201022}
```

Comment:
I'm not sure this is correct. 

I think I either need to sync the existing 20201019 baseline to 20201022 on wcoss-cray or I need to switch this back to 20201019.

Reply:
I'd suggest to make baseline consistent, it would be good to sync the existing 20201019 baseline to 20201022 on wcoss-cray as we then have develop-20201022 for this code update on all the platforms.

Reply:
I've copied the develop-20201019 baseline on wcoss-cray to develop-20201022